drv/stm32h7-spi-server-core/Cargo.toml

```diff
@@ -14,6 +14,7 @@ zerocopy = { workspace = true }
 drv-spi-api = { path = "../spi-api" }
 drv-stm32h7-spi = { path = "../stm32h7-spi" }
 drv-stm32xx-sys-api = { path = "../stm32xx-sys-api" }
+counters = { path = "../../lib/counters" }
```

Comment:
Same here, these changes (and the one to `stm32h7-spi-server-core/src/lib.rs`) seem accidentally included in the PR

Reply:
Those were the result of me trying to rebase without having to do a push --force. The point there being that it would not invalidate previous comments from github. I've done a normal rebase and forced that. These should go away but I will give it a look.